app/shared/components/Footer/DonationButton/DonationButton.tsx

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import Button from '../../design-system/all-components/button/Button';
+import { SvgImage } from '../../svg-image/SvgImage';
```

Comment:
```suggestion
import Button from '~/shared/components/design-system/all-components/button/Button';
import { SvgImage } from '~/shared/components/svg-image/SvgImage';
```

Maybe it would be better to type with '~' ?

Reply:
fixed.